Makefile

```diff
@@ -28,6 +28,7 @@ watch: clean
 			--ignore='**venv' \
 			--ignore='**.github' \
 			--ignore='*.egg-info' \
+			--ignore='**_includes/**/*.txt' \
```

Comment:
To avoid the `make watch` command from refreshing multiple times as the `output.txt` and other workflow outputs are written to this `_includes` folder during the build.